tests/image.test.sh

```diff
@@ -110,7 +110,7 @@ function run_test {
 
   git clone https://github.com/Kong/kong.git || true
   pushd kong
-  git checkout $version_given
+  git checkout $version_given || git checkout next
```

Comment:
if we do not have a version, shouldn't we use a commit-id? if another PR gets merged before this runs, it will checkout the wrong code

Reply:
where would the commit id come from? The docker-kong CI runs fully isolated therefore it only has availbe to it the information that exists in git://kong/docker-kong and the bintray asset itself

Reply:
Can't we pass info along with the bintray asset? a metadata file or something similar?

Reply:
Possibly. I've long thought Kong should have the equivalent of a `package.lock` on the file system that indicates Kong version, git commit, etc.

In lieu of that for official releases everything git://docker-kong has available is what we have on bintray ie https://bintray.com/kong/kong-deb/debian/2.1.4#files

Unofficial builds we have more flexibility ....

Reply:
I think that would  do, though it would be nice to have the same info available on all builds.

Anything you need from me to help implement this?

Reply:
I believe this should accomplish revision for our unofficial images

https://github.com/Kong/kong-build-tools/pull/342